frontend/packages/topology/src/components/export-app/ExportApplication.tsx

```diff
@@ -10,9 +10,14 @@ import { handleExportApplication } from './ExportApplicationModal';
 type ExportApplicationProps = {
   namespace: string;
   isDisabled: boolean;
+  setIsExportAppEnabled?: (showExportAppBtn: boolean) => void;
 };
 
-const ExportApplication: React.FC<ExportApplicationProps> = ({ namespace, isDisabled }) => {
+const ExportApplication: React.FC<ExportApplicationProps> = ({
+  namespace,
+  isDisabled,
+  setIsExportAppEnabled,
+}) => {
```

Comment:
can we avoid this and directly make use of hook `useFlag(ALLOW_EXPORT_APP) in the caller to determine if the export is present

Reply:
updated